### PR TITLE
Add new module: Pear

### LINKED ISSUE #1748: new module: pear
### Is there an existing module for this?

- [X] I have searched for the existing module

### Is there an open PR for this?

- [X] I have searched for existing PRs

### Is there an open issue for this?

- [X] I have searched for existing issues

### Are you going to work on this?

- [X] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

### PR DESCRIPTION
**Pull request by mirpedrol**
## PR checklist

Closes #1748 

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/pear/meta.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test' ]
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+  - assembled:
+      type: file
+      description: FastQ file containing aAssembled .
```
Comment (nvnieuwk) [inline modules/pear/meta.yml:38]: I think something went wrong while typing this :p

**Review by nvnieuwk — APPROVED** [review]
LGTM!

**Comment by nvnieuwk** [comment]
Although it seems conda is failing, do you know why it could be failing? (it seems the checksums are different)

**Comment by mirpedrol** [comment]
Thanks for the review!
Conda is sometime failing because of different md5sums in some modules, I think it's fine to ignore it as long as the Docker and Singularity tests pass :)

**Pull request merged**